src/Draco.LanguageServer/Handlers/DracoFormattingHandler.cs

```diff
@@ -0,0 +1,42 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Draco.Compiler.Api.Syntax;
+using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
+using OmniSharp.Extensions.LanguageServer.Protocol.Document;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Draco.LanguageServer.Handlers;
+
+internal sealed class DracoFormattingHandler : DocumentFormattingHandlerBase
```

Comment:
By the convention - how we started naming handlers -, this should be `DracoDocumentFormattingHandler`